### ISSUE TITLE
Prevent Devtools from automatically switch instances

### ISSUE THREAD

**Issue by OClement** [zalmoxisus/redux-devtools-extension#630]
When multiple stores are available across tabs/windows, the extension tends to switch between them automatically whenever a state change (or action fires, it is not clear which is the trigger), despite explicitly selecting a specific instance.

In some cases this will cause confusions as to which store we're looking at if we don't realize it was switched. Some apps might periodically fire actions causing multiple switching happening in close succession, making the tools almost impossible to use.

**Comment by Empowerful** [comment]
Strong agreement—it's impossible for me to use the tool to debug an instance when it keeps switching automatically to a different one.

**Comment by JerryDoubleU** [comment]
Absolutely, we have a page which runs another page in an iframe, both use NgRx, developing is really painful with that automatic switching. Has anyone find a way to stop that?

**Comment by Empowerful** [comment]
No way to stop it yet but there are [workarounds](https://github.com/zalmoxisus/redux-devtools-extension/issues/610#issuecomment-466745587).

**Comment by djkalmyk** [comment]
is it possible also to have a filter for redux instances?

**Comment by aprilandjan** [comment]
I'm working on an electron app which contains other sites as webview when init. The redux devtools extension change instances so frequently, and finally stop at  `No store found. Most likely you did not allow access to file URLs.` and I am not able to switch manually. Hope there's some option to disable that 😢
